Bio/SeqFeature.py

```diff
@@ -909,7 +909,7 @@ def fromstring(text, length=None, circular=False):
         except ValueError:
             pass
         else:
-            if 0 <= s <= e:
```

Comment:
@peterjc this also had to be changed. I don't think it breaks anything, but flagging in case you can think of something

Reply:
I thought initially it might break single base locations, but this is after the start - 1 has been applied, so I think it is OK.